### PR TITLE
Req 32 data scaffold

### PR DESCRIPTION
**Pull request by holingpoon**
I added two new StreamData classes based on Greg's suggested JSON output per Slack conversation. Let me know if I'm on the right track. Thanks!

### CODE REVIEW

**Comment by rhernand3z** [comment]
Looks good for an initial start. Some variables set in the Class seem to be missing PHPDoc comment blocks. Not sure if that is setup as part of your repo, an example where we used it here -- https://bitbucket.org/NYPL/oauth-server/src/1613a8556bf73d6a1f8320018a742d6e980dd373/src/?at=master

File: src/Model/DataModel/StreamData/ProcessedHoldRequest.php [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+ * @package NYPL\Services\Model\DataModel\StreamData
+ */
+class ProcessedHoldRequest extends StreamData
+{
+    /**
+     * Id associated with a processed hold request.
+     *
+     * @var string
+     */
+    public $requestId = '';
```
Comment (gkallenberg) [inline src/Model/DataModel/StreamData/ProcessedHoldRequest.php]: Change to $id.

File: src/Model/DataModel/StreamData/ProcessedHoldRequest.php [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+
+    /**
+     * Tracking Id associated with a processed hold request.
+     *
+     * @var string
+     */
+    public $trackingId = '';
```
Comment (gkallenberg) [inline src/Model/DataModel/StreamData/ProcessedHoldRequest.php]: Change to $jobId.

File: src/Model/DataModel/StreamData/ProcessedHoldRequest.php [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+
+    /**
+     * Id of patron who put in a hold request.
+     *
+     * @var string
+     */
+    public $patronId = '';
```
Comment (gkallenberg) [inline src/Model/DataModel/StreamData/ProcessedHoldRequest.php]: Change to $patron.

File: src/Model/DataModel/StreamData/ProcessedHoldRequest.php [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
+     * @var string
+     */
+    public $recordType = '';
+
+    /**
+     * Record number requested in a hold request.
+     *
+     * @var string
+     */
+    public $recordNumber = '';
```
Comment (gkallenberg) [inline src/Model/DataModel/StreamData/ProcessedHoldRequest.php]: Change to $record.

File: src/Model/DataModel/StreamData/ProcessedHoldRequest.php [hunk 5]
```diff
@@ -0,0 +1,363 @@
...
+     * Document delivery data represented in an Electronic Document Delivery (EDD).
+     *
+     * @var DocDeliveryData
+     */
+    public $docDeliveryData;
+
+    /**
+     * @return string
+     */
+    public function getRequestId(): string
```
Comment (gkallenberg) [inline src/Model/DataModel/StreamData/ProcessedHoldRequest.php]: Change to $id for getters and setters.

File: src/Model/DataModel/StreamData/ProcessedHoldRequest.php [hunk 6]
```diff
@@ -0,0 +1,363 @@
...
+     */
+    public function setRequestId(string $requestId)
+    {
+        $this->requestId = $requestId;
+    }
+
+    /**
+     * @return string
+     */
+    public function getTrackingId(): string
```
Comment (gkallenberg) [inline src/Model/DataModel/StreamData/ProcessedHoldRequest.php]: Change to $jobId for getters and setters.

File: src/Model/DataModel/StreamData/ProcessedHoldRequest.php [hunk 7]
```diff
@@ -0,0 +1,363 @@
...
+     */
+    public function setTrackingId(string $trackingId)
+    {
+        $this->trackingId = $trackingId;
+    }
+
+    /**
+     * @return string
+     */
+    public function getPatronId(): string
```
Comment (gkallenberg) [inline src/Model/DataModel/StreamData/ProcessedHoldRequest.php]: Change to $patron for getters and setters.

File: src/Model/DataModel/StreamData/ProcessedHoldRequest.php [hunk 8]
```diff
@@ -0,0 +1,363 @@
...
+     */
+    public function setRecordType(string $recordType)
+    {
+        $this->recordType = $recordType;
+    }
+
+    /**
+     * @return string
+     */
+    public function getRecordNumber(): string
```
Comment (gkallenberg) [inline src/Model/DataModel/StreamData/ProcessedHoldRequest.php]: Change to $record for getters and setters.

**Review by gkallenberg — COMMENTED** [review]
Changes needed for model elements that were changed, plus the addition of the $deliveryLocation element, I think for now.

Check the Schema API output for the HoldRequestService stream.

**Review by gkallenberg — APPROVED** [review]
Looks good.

**Pull request merged**